src/Top.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Top.css';
+
+const Top = ({className, owner, name}) => {
+  return (
+    <div className={className}>
+      <div className="top-frame_ly">
+        <div className='user_ly'>
+          <div className='avatar_ly'>
+            <img className="avatar-img" src={owner.profileImageSource} alt="아바타 이미지"/>
```

Comment:
먼저 alt 속성에는 `이미지`라는 단어는 딱히 안넣어주셔도 괜찮아요~

Reply:
추가적으로 서버에서 받아온 데이터로 이미지를 표현해줄수도 있을 것 같아요~
```suggestion
            <img className="avatar-img" src={owner.profileImageSource} alt={`${owner.name} 아바타`}/>
```